pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -71,18 +82,45 @@ func (q *QueryReconciler) Reconcile(ctx context.Context, req *ctrl.Request) (ctr
 		return res, err
 	}
 
-	return ctrl.Result{}, q.runListRestorePoints(ctx, req)
+	finder := iter.MakeSubclusterFinder(q.VRec.Client, q.Vdb)
+	pods, err := finder.FindPods(ctx, iter.FindExisting)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+
+	// find a pod to execute the vclusterops API
+	var podIP string
+	for i := range pods.Items {
+		if i != names.GetNMAContainerIndex() {
+			continue
+		}
+		pod := &pods.Items[i]
+		podIP = pod.Status.PodIP
+		break
+	}
+
+	// setup dispatcher for vclusterops API
+	passwd, err := q.GetSuperuserPassword(ctx, q.Log, q.Vdb)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	prunner := cmds.MakeClusterPodRunner(q.Log, q.VRec.Cfg, q.Vdb.GetVerticaUser(), passwd)
+	dispatcher := q.makeDispatcher(q.Log, q.Vdb, prunner, passwd)
+
+	// extract out the communal and config information to pass down to the vclusterops API.
+	opts := []restorepointsquery.Option{}
+	opts = append(opts,
+		restorepointsquery.WithInitiator(q.Vrpq.ExtractNamespacedName(), podIP),
+		restorepointsquery.WithCommunalPath(q.Vdb.GetCommunalPath()),
+		restorepointsquery.WithConfigurationParams(q.ConfigurationParams.GetMap()),
+	)
+	return ctrl.Result{}, q.runShowRestorePoints(ctx, req, dispatcher, opts)
 }
 
 // fetch the VerticaDB and collect access information to the communal storage for the VerticaRestorePointsQuery CR,
 func (q *QueryReconciler) collectInfoFromVdb(ctx context.Context) (ctrl.Result, error) {
 	vdb := &v1.VerticaDB{}
 	res := ctrl.Result{}
-
-	opts := []restorepointsquery.Option{
-		restorepointsquery.WithInitiator(q.InitiatorPod, q.InitiatorPodIP),
-	}
-
 	err := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
I may have missed this before, but we only need the retry for updates. We are just fetching vdb state. So, I think we need the `RetryOnConflict`.